src/AppBundle/PullRequests/BodyParser.php

```diff
@@ -142,11 +142,13 @@ public function isARefacto()
     }
 
     /**
+     * @Assert\NotBlank(message = "Each pull request should have a ticket linked. Please link it to an existing issue or create a new issue so that it can be reviewed and tested.")
```

Comment:
```suggestion
     * @Assert\NotBlank(message = "Each pull request should have a ticket linked. Please link it to an existing issue or [create a new issue](https://github.com/PrestaShop/PrestaShop/issues/new/choose) so that it can be reviewed and tested.")
```